src/app/+display/section/section.component.html

```diff
@@ -1,13 +1,13 @@
 <div class="card-header font-weight-bold text-primary text-capitalize">
-  <span>{{section.name}}</span>
+  <span>{{ section.name }}</span>
   <div class="float-right" *ngIf="section.shared">
     <div class="embed-dropdown d-inline-block" placement="bottom-right" ngbDropdown>
-      <i class="fa fa-lg fa-share-alt" aria-hidden="true" ngbDropdownToggle></i>
+      <i class="fa fa-lg fa-share-alt" ngbDropdownToggle ></i>
       <div class="dropdown-menu" ngbDropdownMenu>
         <div class="form-group col-12">
-          <label>{{ 'DISPLAY.EMBEDDABLE.LABEL' | translate }}</label>
+          <label for="embeddedDropdown1">{{ "DISPLAY.EMBEDDABLE.LABEL" | translate }}</label>
```

Comment:
I still have concerns with the potential non-distinct `id` due to the possibility of multiple section components being renderred.